src/utils/git/getLatestRelease.ts

```diff
@@ -1,14 +1,20 @@
 import * as semver from 'semver'
 import { getTag, TagPointer } from './getTag'
 
+function isValid(tag: string) {
```

Comment:
Looks like we can drop `isValid` as nothing is using it :)

Reply:
yup, sorry about that, distracted by real work, also the suggestion had a few typos too so I cleaned those up as well. we should be good to go now.